### DISCUSSION TITLE
The addon keeps disappearing from the Firefox Fenix (Android) context menu

### DISCUSSION THREAD

**Discussion by ghost** [gildas-lormeau/SingleFile#1564] [Q&A]
The icon frequently disappears from the 'extensions' menu, and from what I understand, pages can't be saved as a result.

![Screenshot_20240913-165638_Firefox](https://github.com/user-attachments/assets/9485028a-0bd6-43ee-bebf-bcf18926630e)

the addon is displayed as enabled in 'extension manager' (the "run in private browsing" toggle is non-functional as well):

![Screenshot_20240913-165720_Firefox](https://github.com/user-attachments/assets/079abed6-affe-4411-93b9-225388bad516)

Reinstalling the extension is the only solution I know, but sometimes fails.

---

`org.mozilla.firefox` version 130.0
Singlefile `1.22.68`

May be a Firefox Fenix bug as well, I may report.

**Comment by gildas-lormeau** [comment]
I was not able to reproduce the issue but I think it's indeed a bug in Fenix.

**Comment by ghost** [comment]
Dark Reader seems to also be equally affected (functions on the background, vanishes from the menu).

I'm closing this discussion. For further reference:

https://support.mozilla.org/en-US/questions/1463572

**Reply by gildas-lormeau** [reply]
Thank you for the additional info!
